test/unit/infura-controller-test.js

```diff
@@ -1,34 +1,58 @@
-// polyfill fetch
-// global.fetch = function () {return Promise.resolve({
-//     json: () => { return Promise.resolve({"mainnet": "ok", "ropsten": "degraded", "kovan": "down", "rinkeby": "ok"}) },
-//   })
-// }
-// const assert = require('assert')
-// const InfuraController = require('../../app/scripts/controllers/infura')
-//
-// describe('infura-controller', function () {
-//   var infuraController
-//
-//   beforeEach(function () {
-//     infuraController = new InfuraController()
-//   })
-//
-//   describe('network status queries', function () {
-//     describe('#checkInfuraNetworkStatus', function () {
-//       it('should return an object reflecting the network statuses', function (done) {
-//         this.timeout(15000)
-//         infuraController.checkInfuraNetworkStatus()
-//           .then(() => {
-//             const networkStatus = infuraController.store.getState().infuraNetworkStatus
-//             assert.equal(Object.keys(networkStatus).length, 4)
-//             assert.equal(networkStatus.mainnet, 'ok')
-//             assert.equal(networkStatus.ropsten, 'degraded')
-//             assert.equal(networkStatus.kovan, 'down')
-//           })
-//           .then(() => done())
-//           .catch(done)
-//
-//       })
-//     })
-//   })
-// })
+const assert = require('assert')
+const InfuraController = require('../../app/scripts/controllers/infura')
+
+describe('infura-controller', function () {
+  var infuraController
+  let response
+
+  before(async function () {
+    infuraController = new InfuraController()
+    response = await infuraController.checkInfuraNetworkStatus()
```

Comment:
I agree, you probably should mock the network response. I know we hit the network in other tests, but that's bad, so it's worth figuring it out and doing it right.